HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # cloudpathlib Changelog
 
+## v0.8.1 (UNRELEASED)
+ - Implemented some simple methods trivially to make paths from unknown sources more usable ([Issue #149](https://github.com/drivendataorg/cloudpathlib/issues/149), [PR #230](https://github.com/drivendataorg/cloudpathlib/pull/230))
```

Comment:
Would you mind listing the added methods so it's easy to find where these ones were added?